clients/src/main/java/org/apache/kafka/common/metrics/QuotaViolationException.java

```diff
@@ -17,15 +17,26 @@
 package org.apache.kafka.common.metrics;
 
 import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.MetricName;
 
 /**
  * Thrown when a sensor records a value that causes a metric to go outside the bounds configured as its quota
  */
 public class QuotaViolationException extends KafkaException {
 
     private static final long serialVersionUID = 1L;
+    public MetricName metricName;
+    public Double value;
+    public Double bound;
```

Comment:
I'm not super-keen on the accessors, it's just the general Kafka convention. I'm keen on the fields being `final` though. Also, do we want `value` and `bound` to be nullable? If not, then they should be lowercase `double`.